README.md

```diff
@@ -22,9 +22,9 @@ See what's new [here](release-notes.md)
 
 ### download
 ```
-wget http://repo1.maven.org/maven2/report/donut/1.1/donut-1.1-one-jar.jar
+wget http://repo1.maven.org/maven2/report/donut/1.2/donut-1.2-one-jar.jar

```

Comment:
Happy to rollback into a separate PR if that helps releasing a new version. Does the release auto bump the version like the maven release plugin? I wasn't sure.

Reply:
It was purely so that we don't confuse users. `1.1`, the currently available version doesn't expect the new format and we agreed to keep the readme updated as per the latest version.